app/models/user.rb

```diff
@@ -11,6 +11,9 @@ class User < ActiveRecord::Base
 
   before_create :setup_role
 
+  # add scope
+  scope :comment_notifiable, ->(conference) {joins(:roles).where('roles.name IN (?)', [:organizer, :cfp]).where('roles.resource_id = ?', conference.id)}
```

Comment:
Even though we don't currently use roles table for roles other than scoped to **Conference**, in my opinion it is still better to fully form our where statement. 
resource_type: 'Conference' and resource_id: conference.id 
or we can just do  resource: conference